dotnet-logging/LogLevelsWithSerilog/LogLevelsWithSerilog/Controllers/HomeController.cs

```diff
@@ -0,0 +1,59 @@
+﻿using LogLevelsWithSerilog.Manager;
+using Microsoft.AspNetCore.Mvc;
+
+namespace LogLevelsWithSerilog.Controllers
```

Comment:
Let's use file-scoped namespaces to reduce nesting

Reply:
I updated my code